### PR TITLE
Doc of Windows precompiled demos - remove "common dlls"

### LINKED ISSUE #7721: Precompiled Polyhedron demo already contains the required DLLs
The zip file of the Polyhedron demo available in the Overview page already contains all the required DLLs. 
And more important: if one extracts `CGAL-demoDLLs.zip` into the folder and thus overwrites the DLLs that are already there, then the demo does not work anymore.

I don't know if it is the same for the other demos.

Note that [this page](https://doc.cgal.org/latest/Manual/windows.html#title15) should be fixed too.

### PR DESCRIPTION
**Pull request by janetournois**
## Summary of Changes

All the dlls are now included in the main zip archive, so let's remove mentions of "common dlls" archive.

## Release Management

* Affected package(s): Doc
* Issue(s) solved (if any): fix #7721

### CODE REVIEW

**Comment by lrineau** [comment]
Jane, the change to `Documentation/doc/resources/1.9.6/BaseDoxyfile.in` should be duplicated to `Documentation/doc/resources/1.8.13/BaseDoxyfile.in`.

**Comment by lrineau** [comment]
It looks good: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.0-Ic-71/output2/Manual/packages.html (not that this link will only lasts a few days: in a week the content will be automatically removed).

File: Documentation/doc/Documentation/windows.txt [hunk 1]
```diff
@@ -276,8 +276,8 @@ dependencies.
 \section sec-win-demo Trying CGAL algorithms using a Precompiled Demo
 \cgal provides a few demos offering a graphical user interface for some \cgal algorithms.
 All precompiled demos are available for download on the \ref packages page.
-Two links are provided in the column on the right: the demo itself (under **Windows Demo**) and a set of dlls (under **Common Demo Dlls**)
-that contains dlls files that must be placed in the same directory as the `.exe` of the demo.
+A link is provided in the column on the right, to download an archive. It contains the demo executable itself,
+and a set of dll files that must be placed in the same directory as the `.exe` of the demo.
```
Comment (sloriot) [inline Documentation/doc/Documentation/windows.txt:280]: if the common dlls are actually shipped together with the exe, then the second part of the sentence can be removed.

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0-Ic-84

**Pull request merged**